packages/styles/scss/utilities/_index.scss

```diff
@@ -10,6 +10,7 @@
 @forward 'component-reset' as component-*;
 @forward 'convert' show em, rem;
 @forward 'focus-outline';
+@forward 'hide-at-breakpoint';
```

Comment:
IIRC we also need to be sure this file is included in the build via `packages/styles/files.js` 

https://github.com/carbon-design-system/carbon/blob/775ab4c132ee42e4ba39b2fbee1d0ce13475a1b7/packages/styles/files.js#L180-L197

Reply:
Good catch 😅